pvlib/pvarray.py

```diff
@@ -296,6 +296,10 @@ def huld(effective_irradiance, temp_mod, pdc0, k=None, cell_type=None):
         P_{dc} = G' P_{dc0} (1 + k'_1 \log(G') + k'_2 \log^2 (G') + k'_3 T' +
                  k'_4 T' \log(G') + k'_5 T' \log^2 (G') + k'_6 T'^2)
 
+
+    Users should be aware that at very low irradiance, i.e.,
+    :math:`G' < 20 W/m^2`, :math:`P_{dc}` may be negative.
```

Comment:
```suggestion
    :math:`G' < 20 W/m^2`, :math:`P_{dc}` may be negative.
    This is caused by the :math:`\log(G')` term, which tends toward 
    negative infinity as irradiance approaches zero.
```
I can provide a reference that discusses this, if it is of interest.